src/melbalabs/summarize_consumes/package.py

```diff
@@ -1,6 +1,6 @@
 import importlib.metadata
 
-VERSION = "2024.1062"
+VERSION = "2024.1063"
```

Comment:
the version also gets updated in pyproject.toml.

It's automated via tasks.py. when you type eg
`inv commit 'some commit msg'`.
I usually do it for my last commit before I publish a new version.